Scripts/ultraschall_set_next_marker_to_planning_stage.lua

```diff
@@ -24,8 +24,44 @@
 ################################################################################
 ]]
  
+local info = debug.getinfo(1,'S');
+script_path = info.source:match[[^@?(.*[\/])[^\/]-$]]
+dofile(script_path .. "ultraschall_helper_functions.lua")
+planned_color = ultraschall.ConvertColor(100,255,0)
 
--- Print Message to console (debugging)
-function Msg(val)
-  reaper.ShowConsoleMsg(tostring(val).."\n")
+function get_position()
+    playstate=reaper.GetPlayState() --0 stop, 1 play, 2 pause, 4 rec possible to combine bits
+    if playstate & 1 == 1 or playstate & 4==4 then
+        return reaper.GetPlayPosition()
+    else
+        return reaper.GetCursorPosition()
+    end
 end
+
+function set_next_marker_to_planning_stage()
+  cursor_pos=get_position()
+  -- iterate over list of all markerz
+  retval, num_markers, num_regions = reaper.CountProjectMarkers(0) -- get number of all markers
+  
+  if num_markers>0 then
+    retval, isrgn, last_pos, rgnend, name, markrgnindexnumber, color = reaper.EnumProjectMarkers3(0, 0) -- get first marker
```

Comment:
Gibt's dafür ne Doku? Wo, wie, was ist "EnumProjetMarkers3"?

Funtionsnamen mit "3" am Ende ... *kopfschüttel*

Reply:
Tja, was soll ich sagen. Hier ist die API "Dokumentation" dieser Funktion:

https://www.cockos.com/reaper/sdk/reascript/reascripthelp.html#EnumProjectMarkers3

Auf der Seite findest du auch alle anderen "dokumentierten" API Funktionen. Mehr haben wir nicht. Das mag man beklagen, ist von uns aber nicht zu ändern und Teil der Herausforderung des Projektes. Wir betreiben hier schon auch reverse Enigeering.

Zwei gute Strategien zu "was soll dieser API Call bloß" sind:
1) im REAPER Forum nach dem Funktionsnamen suchen
2) auf GitHub nach dem Funktionsnamen suchen. Es gibt doch erstaunlich viel LUA Code zu  REAPER hier, und das nicht von uns. Aus dem Code heraus wird oft klarer, was da wie gemeint ist.

Reply:
Es gibt von EnumProjectMarkers mehrere Funktionsversionen, also auch eine ohne Zahl und eine mit ner 2 am Ende.

Reply:
Mach die Doku doch auch hier auf github. dann könne alle Reaper-/Lua-Enthusiasten etwas dazu beitragen... just my 2,78 ct

Reply:
Das ist in meinem Dokusystem geschrieben, das noch nicht dokumentiert ist XD